### DISCUSSION TITLE
Custom fields vs. change of base OM datamodel for external facing attirbutes

### DISCUSSION THREAD

**Discussion by geoHeil** [open-metadata/OpenMetadata#9843] [General]
I am thinking how to best add the following types of attributes to OM and wonder if these should be custom attributes or if OM community can help to standardize.

I will try to cluster them by topic:

## dataset specific

- spatial extent
- temporal extent
- semantic type descriptions like in metabase (geospatial ZIP code) i.e. person/e-mail/... 
- other attributes listed in Google Dataset search (https://developers.google.com/search/docs/appearance/structured-data/dataset )  this specification when describing a dataset
- additional quality KPI: Whylogs: frequent strings, medians ,... 

## external

- license of the data
- owner vs.  provider vs. funder of a study which provided the data (i.e. healthcare study data)
- URI of external source system
- update  frequency i.e. OGD data might outline a day or period (monthly, yearly, specific date) for the next update to happen
- flag if the data is stored in the own warehouse or only referenced from a remote source

**Comment by pmbrull** [comment]
cc @harshach @sureshms if you'd like to chip in.

My thoughts here would be that some of the points can be added as metrics for the profiler to compute:
- spatial extent
- temporal xtent
- additional quality KPI: frequent strings, medians,...

Not sure if its something generic enough to be brought to the top level of an Entity
